app/javascript/crayons/MultiInput/__stories__/MultiInput.stories.jsx

```diff
@@ -0,0 +1,31 @@
+import { h } from 'preact';
+import { MultiInput } from '../MultiInput';
+import MultiInputDoc from './MultiInput.mdx';
+
+export default {
+  title: 'BETA/MultiInput',
+  argTypes: {},
+  parameters: {
+    docs: {
+      page: MultiInputDoc,
+    },
+    argTypes: {
+      placeholder: {
+        description:
+          'Placeholder text, shown when no selections have been made yet',
+      },
+    },
+  },
+};
+
+export const Default = (args) => {
+  return <MultiInput {...args} />;
+};
+
+Default.args = {
+  placeholder: 'Add an email address...',
```

Comment:
In the designs we do not have a placeholder "Add an email address..." but I thought it may be needed for accessibility reasons and also to make the UI easier to understand. Looking for thoughts here.

Reply:
I like the addition of this for accessibility purposes (it looks like we also do this for the tag autocomplete). I also think that "Add an email address..." is consistent with what we have elsewhere!